### PR TITLE
Fix CHANGELOG for release 0.13.0

### PR DESCRIPTION
**Pull request by frankenjoe**

### CODE REVIEW

**Comment by frankenjoe** [comment]
Grr, forgot those bullet points. Already released 0.13.0, though :)

File: CHANGELOG.rst [hunk 1]
```diff
@@ -10,14 +10,14 @@ and this project adheres to `Semantic Versioning`_.
...
-Changed: clean up test requirements
-Changed: improve documentation on time and date data types
-Changed: improve documentation on language Database argument
-Changed: require ``pandas < 1.4.0``
-Fixed: remove deprecated pandas code
+* Changed: ``audformat.utils.union()`` no longer sorts levels
+* Changed: ``audformat.Table.save()`` forces pickle format 4
+* Changed: ``audformat.Table.save()`` loads table before saving it to CSV
+* Changed: clean up test requirements
+* Changed: improve documentation on time and date data types
```
Comment (hagenw) [inline CHANGELOG.rst:17]: This was released with `0.12.4`

File: CHANGELOG.rst [hunk 2]
```diff
@@ -10,14 +10,14 @@ and this project adheres to `Semantic Versioning`_.
...
+* Changed: improve documentation on language Database argument
```
Comment (hagenw) [inline CHANGELOG.rst:18]: This was released with `0.12.4`

File: CHANGELOG.rst [hunk 3]
```diff
@@ -10,14 +10,14 @@ and this project adheres to `Semantic Versioning`_.
...
+* Changed: require ``pandas < 1.4.0``
+* Fixed: remove deprecated pandas code
```
Comment (hagenw) [inline CHANGELOG.rst:20]: This was released with `0.12.4`

File: CHANGELOG.rst [hunk 4]
```diff
@@ -10,14 +10,14 @@ and this project adheres to `Semantic Versioning`_.
...
-Changed: ``audformat.Table.save()`` forces pickle format 4
-Changed: ``audformat.Table.save()`` loads table before saving it to CSV
-Changed: clean up test requirements
-Changed: improve documentation on time and date data types
-Changed: improve documentation on language Database argument
-Changed: require ``pandas < 1.4.0``
-Fixed: remove deprecated pandas code
+* Changed: ``audformat.utils.union()`` no longer sorts levels
+* Changed: ``audformat.Table.save()`` forces pickle format 4
+* Changed: ``audformat.Table.save()`` loads table before saving it to CSV
```
Comment (hagenw) [inline CHANGELOG.rst:15]: This was released with `0.12.4`

**Comment by frankenjoe** [comment]
Yeah, I got a bit too far in the change log :) Should be fixed now.

**Comment by hagenw** [comment]
Yes, looks good now:

image

**Pull request merged**